CHANGELOG

```diff
@@ -8,53 +8,61 @@
 
                               http://glvis.org
 
-Version 3.4.1 (development)
-===========================
 
-- Overhauled rendering and window management backend, replacing X Windows with
-  SDL for platform-native window and event handling. This allows for OpenGL 3+
-  and HiDPI support on Mac OS X.
+Version 4.0, released on Dec 11, 2020
+=====================================
+
+Starting with this version, the GLVis open source license is changed to BSD-3.
+
+Unlike previous GLVis releases, this version requires a C++11 compiler.
+
+- Major overhaul and modernization of the GLVis rendering and window management,
+  replacing X Windows with SDL for platform-native window and event handling.
+  This enables OpenGL 3+ support and HiDPI support on Mac OS X.
 
 - Two rendering backends are included: one for legacy OpenGL contexts without
   support for shaders, and one with full support for modern OpenGL 3 features.
   The modern OpenGL context is preferred by default; a new command-line argument
-  "-oldgl" can be used to only request the legacy backend.
-
-- Transparency and printing in textured coloring modes is now supported, and
-  no-texture coloring has been removed as a result.
-
-- Text on screen is now rendered using a texture atlas.
+  "-oldgl" can be used to request the legacy backend.
 
 - Preliminary support for building GLVis to Javascript/WebAssembly using
-  Emscripten.
+  Emscripten, see https://github.com/GLVis/glvis-jupyter.
+
+- Documented project workflow and provided contribution guidelines in the new
+  top-level file, CONTRIBUTING.md.
 
 - Added several perceptually uniform colormaps "turbo", "viridis", "plasma",
   "fusion", "iceburn", "viola", "pride" and "ocean" from
   * Google AI: https://ai.googleblog.com/2019/08/turbo-improved-rainbow-colormap-for.html
   * Matplotlib: https://bids.github.io/colormap/
   * CMasher: https://github.com/1313e/CMasher
 
-- Added the ability to discretize a palette, i.e. to use just a prescribed
-  number of its colors, see the new "number of colors" input of the F6 key.
+- Added support for visualization of the element ordering curve with 'Ctrl+o'.
+
+- Improved the cutting plane algorithm for curved elements. The key 'I' can be
+  used to switch to the previous (faster) algorithm which is suitable for meshes
+  with planar faces.
+
+- Updated to support the display and slicing of meshes with wedge elements.
+
+- Improved the opening of parallel meshes/solutions.
+
+- Transparency and printing in textured coloring modes is now supported, and
+  no-texture coloring has been removed as a result.
+
+- Text on screen is now rendered using a texture atlas.
```

Comment:
We could probably remove this point, unless it's to mention that freetype is now a required dependency

Reply:
How about

```
- FreeType is now a required dependency and text on screen is rendered using a
  texture atlas.
```

Reply:
That sounds good to me